system/security/plugins/htpasswdSecurity/htpasswdSecurity.cpp

```diff
@@ -112,6 +112,47 @@ class CHtpasswdSecurityManager : public CBaseSecurityManager
 		return authmap;
 	}
 
+    IAuthMap * createFeatureMap(IPropertyTree * authconfig)
+    {
+	    CAuthMap* feature_authmap = new CAuthMap(this);
+
+	    IPropertyTreeIterator *feature_iter = NULL;
+	    feature_iter = authconfig->getElements(".//Feature");
```

Comment:
Why not use 'Owned<IPropertyTreeIterator> it = tree->getElements(".//Feature");'? You may use 'ForEach(*it)' after that. You will not have to check NULL and isValid.